src/platform/cmx/user/syscall.h

```diff
@@ -0,0 +1,51 @@
+/* temporary */
+static inline
+int call_cap_asm(u32_t cap_no, u32_t op, int arg1, int arg2, int arg3, int arg4)
```

Comment:
Is the `src/platform/cmx/user/` directory copied to `components/include/`?  That might be a good solution.